nx-X11/programs/Xserver/hw/nxagent/Options.h

```diff
@@ -384,6 +384,9 @@ typedef struct _AgentOptions
 
   /*
    * Inhibit some XKEYBOARD requests.
+   * 0 = allow all XKEYBOARD requests
+   * 1 = forbid all XKEYBOARD requests if keyboard=clone or keyboard=query
```

Comment:
Again I am not sure what you mean. Have look at nxagentInitXkbWrapper().

If `InihibitXkb` is `0` no own filter wrapper is installed. Which effectively means that all requests are accepted.

If `InihibitXkb` is 1 (the default) all _writing_ XKEYBOARD requests are silently ignored, the others are accepted. 

If `InihibitXkb` is 2 the behaviour is like 1 but only if neither query nor clone are used. If so, the behaviour is like 0.

So my description is wrong, I have improved that.

However, I have just tested again and found that 1 is only working for clone and query (as I wrote in the code) despite being the default. This needs further investigation.